goal_src/jak1/engine/ui/hud.gc

```diff
@@ -64,6 +64,21 @@
                   (-> obj first-init)
                   )
               )
```

Comment:
`(cpad-hold? 0 l2)` / `(cpad-hold? 0 r2)` checks for keeping HUD up are a few lines up from here^

Reply:
The check that prevents the HUD from being pulled up is the `(not *progress-process*)`.

Reply:
Right yeah, just wanted to show that this is now behind the same barrier as L2/R2 triggering the HUD in the first place. (also no longer needed the calls to update `trigger-time`)